backend/testUtils/users.ts

```diff
@@ -1,9 +1,20 @@
 import { UserDTO } from "../types";
 
+// set up test users to return from mock user service
 export const mockAdmin: UserDTO = {
   id: "62c248c0f79d6c3c9ebbea94",
   firstName: "Admin",
   lastName: "One",
   email: "admin@gmail.com",
   role: "Admin",
 };
+
+export const mockTeacher: UserDTO = {
```

Comment:
you might have some merge conflicts here, since I had to export `mockAdmin` as default to please the linter